CHANGELOG.md

```diff
@@ -207,6 +207,22 @@ If there are links back to radios or checkboxes components in your error summary
 
 [Pull request #1426: Make radios and checkboxes components easier to link to from error summary](https://github.com/alphagov/govuk-frontend/pull/1426)
 
+### Update the markup for tabs
```

Comment:
Would this heading still be correct?

```suggestion
### Update the classes in tabs
```

Reply:
Or 'Update tab classes'

Reply:
Sort of… the class has also moved onto another element which makes it slightly more than just updating classes? 🤔

Reply:
Ah yes, good point. How about just 'Update tabs'? Or stick with the original - this one's definitely not a dealbreaker.

Reply:
In which case I think I'll leave it as-is 👍